app/src/main/res/layout/list_item.xml

```diff
@@ -2,34 +2,34 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:background="#fafafa"
```

Comment:
Use resources for colors and strings without hardcoding them

Reply:
@CloudyPadmal  I've made the changes as per requested.